src/screen/ConnectionRetryScreen.tsx

```diff
@@ -34,14 +35,17 @@ export function ConnectionRetryScreen({navigation}: PropTypes) {
 
   return (
     <Layout style={styles.container}>
-      <ScreenNavigation
-        renderTitle={() => (
-          <View style={styles.networkName}>
-            <Text category="s1">Litentry</Text>
-            {currentNetwork ? <NetworkItem item={currentNetwork} isConnected={false} /> : null}
-          </View>
-        )}
-      />
+      <AppHeader>
+        <AppBar.Action icon="" tvParallaxProperties={undefined} hasTVPreferredFocus={false} onPress={noop} />
```

Comment:
Do we need to pass all these props?

Reply:
There is some sort of ts issue going on here. bad typings in repaper I suppose. it will error without this tv props.

Reply:
sorry, it's already fixed https://github.com/callstack/react-native-paper/issues/2965